assets/router/service-cloud.yaml

```diff
@@ -5,6 +5,8 @@ apiVersion: v1
 metadata:
   # Name is set at runtime.
   namespace: openshift-ingress
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
```

Comment:
This service is created for any cloud provider environments (AWS, GCP, etc.) but the new annotation is specific to AWS (or the annotation name is misleading?)
May be we could inject the annotation only when the platform is AWS?  Similar to what we do here: https://github.com/openshift/cluster-ingress-operator/blob/1c6c23eff519eb40089c59938b09807897149bf4/cmd/cluster-ingress-operator/main.go#L102

Reply:
Yeah, the service and the deployment both need configured as a unit when we're using a cloud HA strategy *and* we're on a supported cloud.